.gitops/charts/traffic-court-dev-values.yaml

```diff
@@ -52,8 +52,8 @@ oracle-data-api:
     tag: "1.55.0"
     pullPolicy: Always
   env:
-    "ORDS_API_OCCAM_URL": "https://dev.jag.gov.bc.ca/ords/devj/occamords/occam"
-    "ORDS_API_TCO_URL": "https://dev.jag.gov.bc.ca/ords/devj/tcoords/tco"
+    "ORDS_API_OCCAM_URL": "https://ords-0198bb-dev.apps.silver.devops.gov.bc.ca/occamords/occam"
+    "ORDS_API_TCO_URL": "https://ords-0198bb-dev.apps.silver.devops.gov.bc.ca/tcoords/tco"
```

Comment:
Not sure this will work, these routes have ip whitelists applied to them. Plus this is a public route, you are basically going out to the internet and back into OpenShift. This endpoint just proxies to ORDs, see config map ords-proxy-configuration

Reply:
undid this change